Run git syncs in transactions

khasinski opened the pull request:
This should prevent the cases in which the sync works only partially resulting in missing data or "blips" in specific repo data. Since there are no tests and I don't have a full local setup I'm not sure how to test it though 😅

khasinski committed:
Run git syncs in explicit transactions

patrickdevivo commented:
Thanks for opening this @khasinski ! The `-1` flag is _supposed_ to execute all of these `-c` commands in a single transaction, but I like how this approach is much more explicit/clearer, so it makes sense to use it.

Do you mind removing the `-1` flags? We're working on some instructions on how to validate these syncs locally, and a test suite as well 🙂 

<img width="1855" alt="image" src="https://user-images.githubusercontent.com/57259/231826679-d0d8eb3b-e7ca-469d-9711-dcfb6c0877e4.png">

khasinski commented:
Yup, I've assumed `-1` means machine-readable format for scripting like `ls -1`, boy I was wrong 😅 

I've removed `-1` from scripts and added `--quiet` for those missing it
